Tests/Invoke-QualityTests.ps1

```diff
@@ -0,0 +1,34 @@
+<#
+.SYNOPSIS
+Test runner for code quality tests
+
+.DESCRIPTION
+Test runner for code quality tests
+
+.EXAMPLE
+Invoke-QualityTests.ps1
+
+.EXAMPLE
+Invoke-QualityTests.ps1
+
+#>
+
+[CmdletBinding()]
+param()
+
+$TestParameters = @{
+    OutputFormat = 'NUnitXml'
+    OutputFile   = "$PSScriptRoot\TEST-Quality.xml"
+    Script       = "$PSScriptRoot\Quality"
+    PassThru     = $True
+    Tag          = "Quality"
+}
+
+# Invoke tests
+$Result = Invoke-Pester @TestParameters
+
+# report failures
+if ($Result.FailedCount -ne 0) { 
+    Write-Error "Pester returned $($result.FailedCount) errors"
+    throw "Pester returned $($result.FailedCount) errors"
```

Comment:
Why do you throw here but not at the same point in equivilent scripts?

Reply:
I'm pretty sure that was accidentally left in when I was testing why I was getting test failures evern though Pester claimed everything was Ok.

Removed.